rostime/include/ros/duration.h

```diff
@@ -90,11 +100,20 @@ class DurationBase
   bool operator>=(const T &rhs) const;
   bool operator<=(const T &rhs) const;
   double toSec() const { return static_cast<double>(sec) + 1e-9*static_cast<double>(nsec); };
-  int64_t toNSec() const {return static_cast<int64_t>(sec)*1000000000ll + static_cast<int64_t>(nsec);  };
```

Comment:
This was `ll` before. Are you sure that `SEC_TO_NSEC` with a single `l` will be appropriate?

Reply:
It should be cast to the larger type in the multiplication, shouldn't it? I can add some more tests...

Reply:
I think you are right. I was just concerned when I saw it. I don't think more tests are necessary.